components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -155,44 +181,54 @@ public function auto_pods() {
 			// get possible pods
 			$the_pods = $this->the_pods();
 
-			// start output array empty
+			// start output array empty.
 			$auto_pods = array();
 
-			// get pods api class
+			// get pods api class.
 			$api = pods_api();
 
-			// loop through each to see if auto templates is enabled
+			// loop through each to see if auto templates is enabled.
 			foreach ( $the_pods as $the_pod => $the_pod_label ) {
 				// get this Pods' data.
 				$pod_data = $api->load_pod( array( 'name' => $the_pod ) );
+				$options  = pods_v( 'options', $pod_data, array() );
+
+				// if auto template is enabled add info about Pod to array.
+				if ( 1 == pods_v( 'pfat_enable', $options ) ) {
+					$type = pods_v( 'type', $pod_data, false, true );
+
+					switch( $type ) {
```

Comment:
* :no_entry_sign: There must be a space between switch and the first parenthesis '('
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after SWITCH keyword; 0 found

<sub>hash: f9cdf9090ca4bafcb56b4a8e7f811743</sub>

Reply:
```suggestion
					switch ( $type ) {
```